src/main/webapp/app/shared/components/organisms/application-detail-card/application-detail-card.component.html

```diff
@@ -0,0 +1,180 @@
+        [class.disabled-link]="!application().applicant?.user?.website"
+        [href]="application().applicant?.user?.website"
+        [attr.aria-disabled]="!application().applicant?.user?.website"
+        [attr.tabindex]="!application().applicant?.user?.website ? -1 : null"
+        [attr.target]="application().applicant?.user?.website ? '_blank' : null"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <div>Website</div>
+        <fa-icon icon="link" class="icon" [class.disabled-icon]="!application().applicant?.user?.linkedinUrl" />
+      </a>
+
+      <a
+        class="link-item"
+        [class.disabled-link]="!application().applicant?.user?.linkedinUrl"
+        [href]="application().applicant?.user?.linkedinUrl"
+        [attr.aria-disabled]="!application().applicant?.user?.linkedinUrl"
+        [attr.tabindex]="!application().applicant?.user?.linkedinUrl ? -1 : null"
+        [attr.target]="application().applicant?.user?.linkedinUrl ? '_blank' : null"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <div>LinkedIn</div>
+        <fa-icon [icon]="['fab', 'linkedin']" class="icon" [class.disabled-icon]="!application().applicant?.user?.linkedinUrl" />
+      </a>
+    </div>
+  </div>
+</ng-template>
+
+<!-- Template: Generic Text Card for Motivation, Skills, etc. -->
+<ng-template #textCardTemplate let-icon="icon" let-name="name" let-text="text">
+  <div class="text-card">
+    <div class="card-row-icon">
+      <fa-icon [icon]="icon" class="icon" />
+      <div class="flex-1 justify-start text-base">{{ name }}</div>
+    </div>
+    <div class="w-full justify-start">
+      {{ text }}
+    </div>
+  </div>
+</ng-template>
+
+<ng-template #infoRow let-name let-val="val">
+  <div class="card-row">
+    <div class="flex-1 text-left">{{ name }}</div>
+    <div class="flex-1 text-right">{{ val }}</div>
+  </div>
+</ng-template>
+
+<!-- Template: Row with Icon (for grades etc.) -->
+<ng-template #infoRowWithIcon let-name let-val="val">
+  <div class="card-row-icon">
+    <div class="flex-1">
+      {{ name }}
+    </div>
+    <div class="flex-1 flex items-center justify-end gap-2.5">
+      <div class="text-right">
+        {{ val }}
+      </div>
+      <div class="transform -rotate-90">
+        <fa-icon icon="arrow-right-arrow-left" class="icon" />
+      </div>
+    </div>
+  </div>
+</ng-template>
+
+<!-- Template: Card Header -->
+<ng-template #card_header let-header="header" let-icon="icon">
+  <div class="h-14 flex self-stretch items-center justify-center gap-2.5 px-4">
+    <fa-icon [icon]="icon" class="icon" />
+    <div class="font-bold text-center">{{ header }}</div>
+  </div>
+</ng-template>
+
+<!-- Education Information Template -->
+<ng-template #educationInfoTemplate>
+  <!-- Bachelor's Information Card -->
+  <div class="card">
+    <ng-container [ngTemplateOutlet]="card_header" [ngTemplateOutletContext]="{ header: 'BSc Information', icon: 'building-columns' }" />
```

Comment:
I would rename this to "Bachelor's" or "Bachelor Degree" or "Bachelor Details" instead of "BSc", since not all Bachelor's are Bachelor of Science
```suggestion
    <ng-container [ngTemplateOutlet]="card_header" [ngTemplateOutletContext]="{ header: 'Bachelor Degree Information', icon: 'building-columns' }" />
```